bin/sqs_cli.py

```diff
@@ -122,6 +122,11 @@ def list_queues(ctx):
 @click.pass_context
 def create(ctx, queue):
     """Create SQS queue."""
+    queue = queue.strip()
+    if not queue:
+        click.echo("Queue name required.")
+        return
+
```

Comment:
This prevents the queue name from being the empty string which ... makes things interesting.